Node/lib/index.js

```diff
@@ -195,12 +195,17 @@ class BotAuthenticator {
                         res.end();
                     }
                     else {
-                        res.status(302);
-                        res.header("Location", `${options.successRedirect}#${encodeURIComponent(magic)}`);
-                        res.send("redirecting...");
-                        res.end();
                         if (this.options.successRedirect && providerId == 'azuread-openidconnect') {
+                            res.status(302);
```

Comment:
You merged with master first, so its hard to see what this code looked like before your change.  Because line 198 definitely changed, but is showing up as not.    Regardless it look fine to me.

Reply:
Also, I dont think you built the code (or ran tests) because only the files you changed is being committed . (This branch has compiled code checked in, so thats how I know.)